helm/aws-load-balancer-controller/templates/deployment.yaml

```
@@ -148,7 +148,7 @@ spec:
         - /controller
         securityContext:
           {{- toYaml .Values.securityContext | nindent 10 }}
-        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
```

I'd suggest to use the schema described [here](https://github.com/giantswarm/schema/blob/main/image/v0.0.1.json)

I've seen that in general we translate `repository` as `registry + name`.

```suggestion
        image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"

```

---

in the config repo we have a different scheme: https://github.com/giantswarm/config/blob/5a40fee0b2521707ac773c9b22b3ee78ba27c7a1/installations/giraffe/config.yaml.patch#L50

---

yeah, but `image.registry` and `global.image.registry` are also there. Not sure where they are coming from.

this is from `argali/pcn01`:
```
$ helm get values -a aws-load-balancer-controller | yq .image
pullPolicy: IfNotPresent
registry: giantswarm-registry.cn-shanghai.cr.aliyuncs.com
repository: quay.io/giantswarm/aws-alb-ingress-controller
tag: v2.4.1

$ helm get values -a aws-load-balancer-controller | yq .global
image:
  registry: giantswarm-registry.cn-shanghai.cr.aliyuncs.com
imageRegistry: giantswarm-registry.cn-shanghai.cr.aliyuncs.com
```